app/helpers/script_helper.rb

```diff
@@ -0,0 +1,17 @@
+require 'set'
+
+# rubocop:disable Rails/HelperInstanceVariable
+module ScriptHelper
+  def javascript_pack_tag_once(name)
+    @scripts ||= Set.new
+    @scripts.add(name)
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def print_javascript_pack_once_tags
```

Comment:
I'd call it `render_` not `print_` because when I see print I think STDOUT?

Reply:
> I'd call it `render_` not `print_` because when I see print I think STDOUT?

Sounds good. Renamed in 748349d.